rllib/tuned_examples/simple_q/cartpole_simpleq_test.py

```diff
@@ -1,3 +1,6 @@
+# Run with:
+# rllib train -f cartpole_simpleq_test.py --type python \
```

Comment:
same here (-f option still supported?).

Reply:
<img width="379" alt="image" src="https://user-images.githubusercontent.com/5839854/199701153-1e4f5eb2-cbdb-447a-a152-85cc7c4a4761.png">

Reply:
yes, of course :D should follow my own conventions...